api/templates/client.mustache

```diff
@@ -60,9 +60,9 @@ type service struct {
 	client *APIClient
 }
 
-// NewAPIClient creates a new API client. Requires a userAgent string describing your application.
+// newAPIClient creates a new API client. Requires a userAgent string describing your application.
 // optionally a custom http.Client to allow for advanced features such as caching.
-func NewAPIClient(cfg *Configuration) *APIClient {
+func newAPIClient(cfg *Configuration) *APIClient {
```

Comment:
@lesam will this change break how Kapacitor is using the CLI? I can submit a PR to fix it after this merges

Reply:
Yes it will break: https://github.com/influxdata/kapacitor/blob/master/cmd/kapacitor/fluxtask.go#L95

Reply:
Reverted this, made the new helper function return `api.Configuration` instead